BUILD.bazel

```diff
@@ -975,16 +975,24 @@ ray_cc_library(
     ],
 )
 
+ray_cc_library(
+    name = "raylet_client_connection_lib",
+    srcs = ["src/ray/raylet_client/raylet_connection.cc"],
+    hdrs = ["src/ray/raylet_client/raylet_connection.h"],
+    linkopts = select({
+        "@platforms//os:windows": [
+        ],
+        "//conditions:default": [
+            "-lpthread",
+        ],
+    }),
+    deps = ["//src/ray/common:network"],
```

Comment:
I think what you need is `//src/ray/common:asio` but not `//src/ray/common:network`

https://github.com/ray-project/ray/blob/f4bfc55379e9d4cac09135d77f25679f6146ebf4/src/ray/raylet_client/raylet_connection.h#L23

Reply:
But the next line includes the "ray/common/client_connection.h" which is in the `//src/ray/common:network`.
https://github.com/ray-project/ray/blob/f4bfc55379e9d4cac09135d77f25679f6146ebf4/src/ray/raylet_client/raylet_connection.h#L24

Reply:
ohoh sorry I mis-read. Let's add `//src/ray/common:asio`

Reply:
okok! added!